src/components/LoginForm.vue

```diff
@@ -57,63 +57,75 @@
 
 <script>
 import { validateMnemonic } from 'bip39'
+import { computed, ref, defineComponent } from 'vue';
+import { useStore } from 'vuex';
 
-export default {
+export default defineComponent({
   props: {
     modelValue: {
       type: String,
       default: ''
     }
   },
   emits: ['login', 'error', 'update:modelValue'],
-  data: () => ({
-    validForm: true,
-    disabledButton: false,
-    showSpinner: false
-  }),
-  computed: {
-    passphrase: {
+  setup(props, { emit }) {
+    const store = useStore()
+    const validForm = ref(false)
+    let disabledButton = ref(false)
```

Comment:
const

Reply:
It doesn't work with const because later in the code there is another assignment to this variable (particularly in the submit function)